platforms/spring-boot/components-starter/camel-rest-swagger-starter/src/main/java/org/apache/camel/component/rest/swagger/springboot/RestSwaggerComponentConfiguration.java

```diff
@@ -82,7 +82,7 @@
      * load swagger.json resource. Note that the host defined on the component
      * and endpoint of this Component should contain the scheme, hostname and
      * optionally the port in the URI syntax (i.e.
-     * https://api.example.com:8080). Can be overridden in endpoint
+     * \https://api.example.com:8080). Can be overridden in endpoint
```

Comment:
This will be regenerated by the build without escape when the suggested change in `core/camel-endpointdsl/src/main/java/org/apache/camel/builder/endpoint/dsl/RestSwaggerEndpointBuilderFactory.java`

Reply:
Okay Mr.Zoran, can you check now?

Reply:
I ran ./mvnw -Pfastinstall install after doing above change, but this also unescaped according to the above change, is it okay?